Dockerfile-dev

```diff
@@ -0,0 +1,26 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/ziplinedev .
```

Comment:
I think this command needs to specify this dockerfile explicitly, since it's not named just Dockerfile.  e.g.
```bash
docker build -t quantopian/ziplinedev -f Dockerfile-dev
```